development/xdebug/xdebug.SlackBuild

```diff
@@ -97,23 +97,6 @@ mkdir -p $EXTENSION_DIR
 # Install xdebug
 make install DESTDIR=$PKG EXTENSION_DIR=$EXTENSION_DIR
 
-# Build and install debugclient
-cd debugclient
-  CFLAGS="$SLKCFLAGS" \
-  CXXFLAGS="$SLKCFLAGS" \
-  ./configure \
-    --prefix=/usr \
-    --libdir=/usr/lib${LIBDIRSUFFIX} \
-    --sysconfdir=/etc \
-    --localstatedir=/var \
-    --mandir=/usr/man \
-    --docdir=/usr/doc/$PRGNAM-$VERSION/debugclient \
-    --build=$ARCH-slackware-linux \
-    --with-libedit
-  make
-  make install DESTDIR=$PKG
-cd ..
-
 mkdir -p $PKG/etc/php.d
 install -m 644 $CWD/xdebug.ini $PKG/etc/php.d/xdebug.ini.new
 sed -i "s/LIBDIR/lib$LIBDIRSUFFIX/g" $PKG/etc/php.d/xdebug.ini.new
```

Comment:
xdebug ships a generated ini file now as part of the source tarball, so we can just take that and append it to the one we put in place here with:

```
sed -e "1,2d" xdebug.ini >> $PKG/etc/php.d/xdebug.ini.new
```

The first two lines from the ini in the sources is the "autogeneration" warning, which applies to making changes in git, but is just confusing for admins, so we can drop that.

Reply:
Done, thank you